main.go

```diff
@@ -560,12 +562,15 @@ func ReadIP6CIDRsFromSubnetFile(path string, CIDRKey string) []ip.IP6Net {
 }
 
 func newTrafficManager(useNftables bool) trafficmngr.TrafficManager {
+	if runtime.GOOS == "windows" {
+		log.Info("Starting flannel in windows mode...")
+		return &windows.WindowsManager{}
+	}
 	if useNftables {
-		log.Info("Starting flannel in nftables mode")
+		log.Info("Starting flannel in nftables mode...")
```

Comment:
Why the three dots?

Reply:
it's just to follow the convention when you put a log message before an operation that can take some time.

Reply:
I did not know about this convention, thanks!